package.json

```diff
@@ -77,8 +77,8 @@
   },
   "dependencies": {
     "@openmrs/esm-api": "^1.0.8",
+    "@openmrs/esm-error-handling": "^1.1.0",
     "@openmrs/react-root-decorator": "^1.0.0",
-    "kremling": "^2.0.1",
```

Comment:
Kremling is a build-time dependency, so it must be in devDependencies (which it already is in 😄 )